src/webgpu/api/validation/encoding/encoder_state.spec.ts

```diff
@@ -17,21 +17,28 @@ TODO:
 `;
 
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
-import { objectEquals } from '../../../../common/util/util.js';
+import { assert, objectEquals, unreachable } from '../../../../common/util/util.js';
 import { ValidationTest } from '../validation_test.js';
 
 class F extends ValidationTest {
+  private static attachmentTexture: GPUTexture | null = null;
+  private static attachmentTextureView: GPUTextureView | null = null;
+
   beginRenderPass(commandEncoder: GPUCommandEncoder): GPURenderPassEncoder {
-    const attachmentTexture = this.device.createTexture({
-      format: 'rgba8unorm',
-      size: { width: 16, height: 16, depthOrArrayLayers: 1 },
-      usage: GPUTextureUsage.RENDER_ATTACHMENT,
-    });
-    this.trackForCleanup(attachmentTexture);
+    if (!F.attachmentTexture) {
+      F.attachmentTexture = this.device.createTexture({
```

Comment:
This createTexture will be caught inside the expectValidationError call. Not a big deal since it should never fail, but in principle not ideal.